linux_docker_resources/Dockerfile-RHEL

```diff
@@ -17,7 +17,24 @@ RUN dnf install \
     net-tools \
     patch \
     python3-bloom \
-    python3-colcon-common-extensions \
+    python3-colcon-bash \
```

Comment:
Looking into the dependencies of `common-extensions` in Ubuntu, there are some colcon packages missing in this list. (argcomplete, cd, etc.). Probably intentioned but double checking.